DEVELOPMENT.md

```diff
@@ -1,109 +1,87 @@
-  - [Giving that personal access token to `npm`](#giving-that-personal-access-token-to-npm)
-- [Installing the Angular command-line interface](#installing-the-angular-command-line-interface)
-- [Setting up Firebase](#setting-up-firebase)
-  - [Installing the command-line interface](#installing-the-command-line-interface)
-  - [Running the emulator](#running-the-emulator)
-
-## Tell `npm` how to talk to GitHub Packages
-
-GitHub Packages is where we keep all of the images for the website, along with some data about flowers and bees. 
-
-For `npm` read from GitHub Packages, you'll need to provide it with a personal access token.
-
-### Generating the personal access token
-
-You can create a personal access token by going to <https://github.com/settings/tokens> and clicking "Generate new token". Under "Scopes", you'll want to check `repo`, `read:packages`, `write:packages`, and `delete:packages`:
-
-![The "New personal access token" page on GitHub, with those four scopes checked.][i:New personal access token]
-
-Then, you can click "Generate token" at the bottom of the page. You'll get a token that looks like `745b0dcc742dedf87c67ff075f65eb87295e205f`. Keep that around.
-
-### Giving that personal access token to `npm`
-
-Next, open the command-line and run
-
-```sh
-npm login --registry=https://npm.pkg.github.com
+- [Running Buzz About](#running-buzz-about)
+  - [Firebase Emulators](#firebase-emulators)
+  - [Development Server](#development-server)
+- [Testing](#testing)
+  - [Running unit tests](#running-unit-tests)
+  - [Running Cypress](#running-cypress)
+  - [Running Firebase tests](#running-firebase-tests)
+- [Additional Angular CLI commands](#additional-angular-cli-commands)
+  - [Linting](#linting)
+  - [Code scaffolding](#code-scaffolding)
+  - [Further help](#further-help)
+
+## Running Buzz About
+
+### Firebase Emulators
+
+To run the Firebase emulators:
+```
+npm run firebase:emulator
 ```
 
- * When it asks for a username, enter your GitHub username.
- * When it asks for a password, paste in the personal access token (***not*** your GitHub password).
- * When it asks for your email, enter your GitHub email. 
-
-Some sticking-points:
-
-1. To paste text into PowerShell, right-click once.
-2. On github.com/settings/tokens, the personal access token has a space after it&mdash;be careful that you don't copy that! :upside_down_face:
-
-![github.com/settings/tokens with the extra space character highlighted][i:The gosh-darn whitespace]
-
-If you run into trouble, you can always fall back to typing the personal access code by hand.
-
-## Installing the Angular command-line interface
-
-Angular provides a CLI with commands for some common operations, like `ng build`, `ng serve`, `ng lint`, and `ng test`. (There's also `ng generate`, which lets you quickly make a new component or service without having to write out the boilerplate every time.)
+You should be able to access the Firebase Emulator Suite page at http://localhost:4000/
```

Comment:
How do you feel about using the word `console` here? I feel like it's a bit more descriptive, in case people didn't already know what the emulator suite page is. :slightly_smiling_face: 

```suggestion
You should be able to access the Firebase Emulator Suite console at http://localhost:4000/
```